sdk/dapp-kit/test/components/WalletProvider.test.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
I moved all of the tests from `useWallet.test.tsx` to here because the core logic we're testing is really associated with the provider and not the `useWallet` hook even though we're using the `renderHook` method to test the provider. The new test to look at here is on line 98 - `auto-connecting to a wallet works successfully`